include/swift/AST/NameLookup.h

```diff
@@ -101,6 +101,9 @@ class UnqualifiedLookup {
     /// This lookup should include results from outside the innermost scope with
     /// results.
     IncludeOuterResults = 0x10,
+    /// Whether to ignore local variables inside bodies of functions, closures
+    /// and top level code.
+    IgnoreLocalVariables = 0x50,
```

Comment:
This isn't a power of 2. Is it supposed to include IncludeOuterResults?

Reply:
Sorry, my bad, I completely ignored the fact that the values above are hex 🤦‍♂️. Fixed now!